### ISSUE TITLE
Edits for Experiment Design Factors

### ISSUE THREAD

**Issue by richardostler** [Rothamsted-Ecoinformatics/eRA#28]
- [x] Beneath the Factors heading, can you add the text: _Factors are the interventions or treatments which vary across the experiment._
- [ ] Can cards be uniformly sized?
- [x] Rename Factor Recombinations to Factor Combinations
- [x] Beneath the heading can you include the text: _Factor Combinations_ are the combination of factors applied to different plots on the experiment.
- [x] I think the Factor Combinations could be displayed in a tabular format. We only need to include the labels, e.g. N1, rather than N1: nitrogen fertilizer exposure, as the factor will have been defined in the previous section and so can be referenced from the label. The Year and notes section should remain. Also FYM is not displaying as factor here (but is shown in the title)

**Comment by nathcast** [comment]
**Can cards be uniformly sized?** If they were cards, they could be uniformaly sized. I have tried cards, but in an accordion, which is the container above, the cards **lose their bottom border.** It is a bug that has been observed in bootstrap  and it is documented as solved. and I have already spend 2 hours looking into it, because at first I thought it was something I did. These are list_groups.  I might look more into it later.
